BestPractices.md

```diff
@@ -1,10 +1,10 @@
-# BEST PRACTICES #
+# BEST PRACTICES
 
 This is Prolific's Swift best practices document.
 
 The purpose of these best practices is to help you keep clean and bug free Swift code that is recommended to follow in your daily work.
 
-## Table Of Contents ##
+## Table Of Contents
```

Comment:
Table Of Contents uses header level 1, not 2 in the main file. Maybe we change this to `#` and also bump everything else up.